AppCenterAuth/AppCenterAuth/MSAuthErrors.h

```diff
@@ -19,7 +19,12 @@ static NSString *const kMSACAuthErrorDomain = MS_APP_CENTER_BASE_DOMAIN @"Auth.E
 /**
  * Error code for Auth.
  */
-NS_ENUM(NSInteger){MSACAuthErrorServiceDisabled = -420000, MSACAuthErrorPreviousSignInRequestInProgress = -420001,
-                   MSACAuthErrorSignInBackgroundOrNotConfigured = -420002, MSACAuthErrorSignInWhenNoConnection = -420003};
+NS_ENUM(NSInteger){// System framework errors
+                   MSACAuthErrorServiceDisabled = 101,
+
+                   // Sign in errors
+                   MSACAuthErrorPreviousSignInRequestInProgress = 201, MSACAuthErrorSignInConfigNotValid = 202,
+                   MSACAuthErrorSignInWhenNoConnection = 203, MSACAuthErrorSignInDownloadConfigFailed = 204,
```

Comment:
`MSACAuthErrorSignInWhenNoConnection` => `MSACAuthErrorNoConnection` and  we can be a bit more generic for this error, the description will be more explicit.